cmd/ddev/cmd/debug-fixcommands.go

```diff
@@ -2,31 +2,19 @@ package cmd
 
 import (
 	"github.com/ddev/ddev/pkg/ddevapp"
-	"github.com/ddev/ddev/pkg/globalconfig"
 	"github.com/ddev/ddev/pkg/util"
 	"github.com/spf13/cobra"
 )
 
-// DebugFixCommandsCmd fixes up custom/shell commands without having to do a ddev start
+// DebugFixCommandsCmd fixes up global container commands without having to do a ddev start
```

Comment:
There isn't even a reason for this `ddev debug fixcommands` to exist any more is there, after this PR?

Reply:
It certainly becomes a less important command, but I think it will be useful to have a command that syncs global commands into the volume (if someone adds or removes one for example) without having to run `ddev start` on a project.